docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -183,47 +183,69 @@ You can create a `kubeconfig` file for the previously created `yugabyte-platform
 
-## Select the Kubernetes service
+## Configure Kubernetes
 
-In the YugabyteDB Anywhere UI, navigate to **Configs > Cloud Provider Configuration > Managed Kubernetes Service** and select one of the Kubernetes service providers using the **Type** field, as per the following illustration:
+To configure Kubernetes providers, navigate to **Configs > Infrastructure > Managed Kubernetes Service**.
 
-![Kubernetes config](/images/ee/k8s-setup/k8s-configure-empty.png)
+This lists all currently configured providers.
 
-## Configure the cloud provider
+To view a provider, select it in the list to display the **Overview**. You can perform the following on a profile configuration:
 
-Continue configuring your Kubernetes provider as follows:
+- To edit the configuration, select **Config Details**, make changes, and click **Apply Changes**. Note that, depending on whether the configuration has been used to create a universe, you can only edit a subset of options.
+- To view the universes created using the profile, select **Universes**.
+- To delete the configuration, click **Actions** and choose **Delete Configuration**. You can only delete configurations that are not in use by a universe.
 
-1. Specify a meaningful name for your configuration.
-2. Choose one of the following ways to specify **Kube Config** for an availability zone:
-    - Specify at **provider level** in the provider form. If specified, this configuration file is used for all availability zones in all regions.
-    - Specify at **zone level** in the region form. This is required for **multi-az** or **multi-region** deployments. If the zone is in a different Kubernetes cluster than YugabyteDB Anywhere, a zone-specific `kubeconfig` file needs to be passed.
-3. In the **Service Account** field, provide the name of the [service account](#service-account) which has necessary access to manage the cluster (see [Create cluster](../../../../deploy/kubernetes/single-zone/oss/helm-chart/#create-cluster)).
-4. In the **Image Registry** field, specify from where to pull the YugabyteDB image. Accept the default setting, unless you are hosting the registry, in which case refer to steps described in [Pull and push YugabyteDB Docker images to private container registry](../../../install-yugabyte-platform/prepare-environment/kubernetes/#pull-and-push-yugabytedb-docker-images-to-private-container-registry).
-5. Use **Pull Secret File** to upload the pull secret to download the image of the Enterprise YugabyteDB that is in a private repository. Your Yugabyte sales representative should have provided this secret.
+To create an Kubernetes provider, click **Create Config** to open the **Create Kubernetes Provider Configuration** page.
 
-## Configure region and zones
+To fill the provider configuration values using the configuration of the same Kubernetes cluster that your instance of YugabyteDB Anywhere is installed on, click **Autofill local cluster config**.
```

Comment:
https://docs.google.com/document/d/1Kcb4TgvQzeCoAVzUSkPkZvsbH0n2cfeq38X22WeJa_8/edit#heading=h.26rq8tm7d9ai

Can we add  reason behind doing this and also perhaps the screenshot from the above document would be useful as well.